index.d.ts

```diff
@@ -3960,6 +4040,7 @@ declare namespace Eris {
   }
 
   export class ThreadChannel extends GuildChannel implements ThreadTextable {
+    appliedTags: string[];
```

Comment:
When you move this to public thread channel, create a generic which checks if the public thread channel belongs in a forum channel or not (we should specify where necessary if it does)